tests/lib/config.js

```diff
@@ -994,20 +1018,12 @@ describe("Config", function() {
             var StubbedConfig = proxyquire("../../lib/config", configDeps);
 
             var configPath = path.resolve(__dirname, "../fixtures/config-extends/package3/.eslintrc"),
-                configHelper = new StubbedConfig({ useEslintrc: true, configFile: configPath, cwd: process.cwd() }),
+                configHelper = new StubbedConfig({ useEslintrc: false, configFile: configPath, cwd: process.cwd() }),
                 expected = {
                     rules: { "quotes": [2, "double"], "eqeqeq": 2, "valid-jsdoc": 0 },
                     env: { "browser": false }
-                };
-
-            // Reason to override this function in this special case is that I dont want it to keep looking into the
-            // chain of parent directries for .eslintrc or package.json file for configs.
-            // If that is allowed then the expected outcome will change based on the machine you are running this test.
-            sinon.stub(configHelper, "findLocalConfigFiles").returns([
-                path.resolve(__dirname, "../fixtures/config-extends/package3/.eslintrc")
-            ]);
-
-            var actual = configHelper.getConfig(configPath);
+                },
+                actual = configHelper.getConfig(configPath);
```

Comment:
No, this one is simply the ones that don't need `useEslintrc` at all as discussed before.